builtin/mainmenu/credits.json

```diff
@@ -75,6 +75,8 @@
 		"Gregory Currie (gregorycu)",
 		"paradust7",
 		"JacobF",
-		"Jeija <jeija@mesecons.net>"
+		"Jeija <jeija@mesecons.net>",
+		"veprogames",
+		"wrrrzr"
```

Comment:
AFCMS, veprogames and wrrrzr are all below the "previous contributor" threshold and should not be in this list.
or did I do something wrong?